build/scripts/find-lint.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
Please always set `-eu` in your bash scripts.

`#!/bin/bash -eu`. This is because bash is a silly sausage. `-e` means if any pipeline ever ends with a non-zero exit code, terminate the script immediately. `-u` means treat unset local variables as errors (so calling `$FOO` without defining `FOO` first produces an error. These flags at least make bash a bit more sane like a normal programming language.

Reply:
If you want to be particularly picky, you should also set `-o pipefail` to make any command in a pipeline return an error status. By default, only the last command will do this.

Reply:
Ah, I actually didn't set `-e` on purpose in case `golangci-lint run` failed and we exited the script without running the `mv` for go.mod and go.sum. However I guess the `|| exit_code=1` already ensures that that won't happen. For this reason I won't add `-o pipefail` though (although TIL!).

I can add `-e` though. I also omitted `-x` as it seemed a bit annoying to print out everything when running it as a dev.